src/types.ts

```diff
@@ -646,6 +648,24 @@ export interface FunctionOptions<
    */
   concurrency?: number | { limit: number };
 
+  /**
+   * batchEvents specifies the batch configuration on when this function
+   * should be invoked when one of the requirements are fulfilled.
+   */
+  batchEvents?: {
+    /**
+     * The maximum number of events to be consumed in one batch
+     */
+    maxSize: number;
```

Comment:
I get what you're saying but it's more often referred as `batch size` instead of `batch limit`. I'd prioritized natural language more than consistency for this one.